packages/eslint-plugin/docs/Testing.md

```diff
@@ -0,0 +1,32 @@
+# Testing
+
+## Introduction
+
+This document provides information about how to test the eslint-plugin after adding or removing rules.
+
+## Example-app
+
+The example-app folder is located at the root of `react-native-project-config`.
+The goal of each file of example-app is to simulate the behavior of a rule or a set of rules, that you can visually check it or by running the `yarn lint` command.
+When you add or remove a rule from the recommended config, please modify the corresponding file in the example-app folder.
```

Comment:
"please modify (or add) the corresponding file" si jamais le test n'existe pas pour une nouvelle règle ?

Reply:
Exact, j'ajoute ça :)